.devcontainer/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM mcr.microsoft.com/vscode/devcontainers/python:3.9-bullseye
+#FROM mcr.microsoft.com/vscode/devcontainers/python:3.9-bullseye
```

Comment:
yes - I will do that in another PR.  The changes are tentative right now because this functionality is sort of in beta right now (try to figure out what's the best setup).  I'm hoping to eventually discuss making it a supported part of the repo and provide a clean setup.